src/catalog/catalog.cpp

```diff
@@ -223,14 +223,21 @@ CollateCatalogEntry *Catalog::GetEntry(ClientContext &context, string schema_nam
 	return (CollateCatalogEntry *)GetEntry(context, CatalogType::COLLATION_ENTRY, move(schema_name), name, if_exists);
 }
 
-void Catalog::AlterTable(ClientContext &context, AlterTableInfo *info) {
+void Catalog::Alter(ClientContext &context, AlterInfo *info) {
+	auto catalog_type = info->GetCatalogType();
```

Comment:
This is my only other nit: we don't need the`catalog_type` unless we're inside of the `info->schema == INVALID_SCHEMA` block.

Thanks so much for letting me review and answering my questions!

Reply:
Fair enough, thanks for the comments!